core/models.py

```diff
@@ -88,7 +88,8 @@ class PacuSession(Base, ModelUpdateMixin):
         'VPC',
         'WAF',
         'Account',
-        'AccountSpend'
+        'AccountSpend',
```

Comment:
Adding new fields will break existing pacu sessions. There are other modules pending so we will have to merge them simultaneously to reduce impact.